Assets/MixedRealityToolkit.Tests/EditModeTests/Core/Extensions/CameraExtensionTests.cs

```diff
@@ -11,6 +12,19 @@ public class CameraExtensionTests
         private static Camera testCamera = null;
         private const float MarginTolerance = 0.005f;
 
+        private class TestPoint
+        {
+            public bool ShouldBeInFOV;
+            public Vector3 Point;
+            public TestPoint(Vector3 point, bool isInFOV)
+            {
+                ShouldBeInFOV = isInFOV;
+                Point = point;
+            }
+        }
+
+        private static List<TestPoint> TestPoints = new List<TestPoint>();
```

Comment:
Nit: Add a comment saying that these points are run with the same expected results for both IsInFOVCone and IsInFOV. Because we had set up some generic-type test data I thought that we might be testing that the two functions have different functionality of the two different functions, but we expect the same results for these two given these settings.